lib/install/management/store_files.go

```diff
@@ -189,3 +191,44 @@ func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfi
 	}
 	return nil
 }
+
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) {
+	if d.force {
```

Comment:
Go idom is to address the short path first and return. This avoids large swathes of indented code.